bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/IOExceptionWithStatus.java

```diff
@@ -36,6 +37,7 @@ public IOExceptionWithStatus(String message, OperationRuntimeException cause) {
   /**
    * Status from the provided OperationRuntimeException.
    */
+  @SuppressWarnings("deprecation")
```

Comment:
Hmm.. why? As I read it, StatusRuntimeException shouldnt trigger deprecation warnings. Which one should be used instead?